standalone/src/models/product-selection/product-selection-draft/presets/index.ts

```diff
@@ -1,6 +1,27 @@
-import empty from './empty';
+import * as empty from './empty';
 import sampleDataB2B from './sample-data-b2b';
 
-const presets = { empty, sampleDataB2B };
-
-export default presets;
+export const restPresets = {
+  empty: empty.restPreset,
+  sampleDataB2B: {
+    usMediumCustomersCatalog: sampleDataB2B.usMediumCustomersCatalog.restPreset,
+    defaultProductSelection: sampleDataB2B.defaultProductSelection.restPreset,
```

Comment:
I wasn't sure here how to go about this nested preset structure
`presets -> sampleDataB2B -> usMediumCustomersCatalog, defaultProductSelection`
is it fine to do it like this?

Reply:
I pushed [this commit](https://github.com/commercetools/test-data/pull/809/commits/604ad75f87da3c09b69c863414ebd8c2d5dd4128) with a refactor for one of the presets. 
Perhaps you can take it as a basis for refactoring the other one.